infra/Pos/StateLock.hs

```diff
@@ -131,10 +134,10 @@ stateLockHelper doWithMVar prio reason action = do
     withPriorityLock prioLock prio $ doWithMVar mvar $ \hh -> do
         timeEndWait <- currentTime
         liftIO . usingLoggerName lname $
-            slmAcquire (timeEndWait - timeBeginWait)
+            slmAcquire reason (timeEndWait - timeBeginWait)
```

Comment:
This is coming from [here](https://github.com/input-output-hk/cardano-sl/commit/6a4c192624e75ece572fab463df8a3143c00cc7d#diff-76bc123843b996823f7c5e36659c28ec), that we backported [here](https://github.com/input-output-hk/cardano-sl/pull/2255/files), so I think we are good.